lib/pf/services/manager/routes.pm

```diff
@@ -131,7 +131,7 @@ sub isAlive {
         }
     }
     my $routes_applied = $FALSE;
-    $routes_applied = defined(pf_run("route | grep ".$route_exist)) if ($route_exist);
+    $routes_applied = defined(pf_run("/sbin/route | grep ".$route_exist)) if ($route_exist);
```

Comment:
Shouldn't be in the debianize.patch ?, on centos it's /usr/sbin/route

Reply:
On CentOS:

```shell
# sudo -u pf -i

-sh-4.2$ ls -rtld /sbin
lrwxrwxrwx. 1 root root 8  1 juin   2019 /sbin -> usr/sbin
```

Behavior will be the same for Debian Buster: https://www.debian.org/releases/buster/amd64/release-notes/ch-whats-new.en.html#merged-usr